src/openms/include/OpenMS/FILTERING/DATAREDUCTION/MassTraceDetection.h

```diff
@@ -114,6 +114,7 @@ namespace OpenMS
 
         // parameter stuff
         double mass_error_ppm_;
+        double mass_error_da_; // added by kyowon jeong
```

Comment:
hehe don't need to sign extra lines. git blame is our friend if we want to know :P